quinn-proto/src/crypto.rs

```diff
@@ -99,11 +99,16 @@ pub fn reset_token_for(key: &SigningKey, id: &ConnectionId) -> [u8; RESET_TOKEN_
     result
 }
 
-#[derive(Clone)]
+pub struct OneRttCrypto {
+    context: CryptoContext,
+    local_secret: Vec<u8>,
+    remote_secret: Vec<u8>,
+}
```

Comment:
I'm not sure this merits having a separate type, since it's harmless for initial to have these fields and either store its secrets in them for consistency, or just ignore them. @djc, do you have an opinion?

Reply:
Yeah, I think it should just be a single type, and store the initial secrets in them.

Reply:
I'll move them to `CryptoContext` and assign them into `new_initial()` as well.

Then removing the `enum` should not be a problem anymore.